app/styles/modules/_notifications.scss

```diff
@@ -0,0 +1,31 @@
+.notification {
+  @extend .row;
+  height: $double-grid;
+  &.dismissable {
+    background: $primary;
+    color: $white;
+    a {
+      color: $white;
+      text-decoration: underline;
+    }
+  }
+  &.static {
+    position: absolute;
+    z-index: 9999;
+    background: $white;
+    box-shadow: 0 1px 1px rgba(0,0,0,.2);
```

Comment:
Not sure if we want to use a variable here. Closest we have in _variables.scss is this gem:
```css
$translucent-black: rgba(0, 0, 0, 0.4);
```